packages/docs/site/docs/01-start-here/02-overview.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Overview
+slug: /overview
+---
+
+# Overview
+WordPress Playground is an online platform that allows you to experiment and learn about WordPress without affecting your live website. It's like a virtual sandbox where you can play around with different features, designs, and settings in a safe and controlled environment.
+
+## Here's how it works
+When you first start using WordPress Playground, you'll be provided with a separate space where you can create and customise your own WordPress website. This space is completely isolated from your actual website, so any changes you make here won't affect your live site.
+
+## Try themes and plugins on the fly
+Within the WordPress Playground, you can explore various themes, which are pre-designed templates that determine the appearance of your website. You can choose from a wide range of themes and see how they look on your site. You can also modify the colors, fonts, layouts, and other visual elements to create a unique design.
+In addition to themes, you can experiment with plugins too. With WordPress Playground, you can install and test different plugins to see how they work and what they can do for your site. This allows you to explore and understand the capabilities of WordPress without worrying about breaking anything.
+
+## Try content on the run
+Another great feature of WordPress Playground is the ability to create and edit content. You can write blog posts, create pages and add media like images & videos to your site. This helps you understand how to organise and structure your content effectively.
```

Comment:
Cool! Maybe add a small screenshot from the editor?

Reply:
I have not mentioned anything about Gutenberg above. Exactly where do you want me to put the screenshot?

Reply:
let's use "images and video" instead of "images & video"

also let's use American English and spell "organize" instead of "organise"

> Exactly where do you want me to put the screenshot?

not sure what he was referencing, but maybe he thought it would be good right here in this spot to show editing inside the playground.

---

I love that this section encourages creating content as a way to learn, but I feel like we should add an additional explanation that it disappears - this is not the place to write an autobiography 😆 

> The content you create is limited to the Playground on your device and disappears once you leave it, so you are free to explore and play without risking breaking any actual site.

My suggested wording here could probably be better.